dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/LambdaEventDAO.java

```diff
@@ -89,7 +89,6 @@ private List<Predicate> processQuery(String filter, String sortCol, String sortO
                 } else {
                     query.orderBy(cb.desc(sortPath), cb.desc(event.get("id")));
                 }
-                predicates.add(sortPath.isNotNull());
```

Comment:
Question, how do nulls get sorted as a result of this change? 
Assuming they're either first or last?

Reply:
Yes, so nulls will be first when sortOrder is `asc` and last when sortOrder is `desc`